docs/source/installation.rst

```diff
@@ -35,4 +35,61 @@ The latest release is |release|.
 
     pip install git+https://github.com/mesosphere/dcos-e2e.git@master
 
+
+Windows
+~~~~~~~
+
+The only supported way to run DC/OS E2E on Windows is using Vagrant and VirtualBox.
+
+- Ensure Virtualization and VT-X support is enabled in your PC's BIOS.
+- Disable Hyper-V virtualisation.
+- Install VirtualBox and VirtualBox Extension Pack.
+- Install Vagrant.
+- Install the Vagrant plugin for persistent disks:
+
+.. code:: ps1
+
+    vagrant plugin install vagrant-persistent-storage
+
+- Optionally install the Vagrant plugins to cache package downloads and keep guest additions updates:
+
+.. code:: ps1
+
+        vagrant plugin install vagrant-cachier
+        vagrant plugin install vagrant-vbguest
+
+- Start Powershell and download the E2E ``Vagrantfile`` to a directory containing a DC/OS installer file:
+
+.. code:: ps1
+
+    ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/mesosphere/dcos-e2e/master/vagrant/Vagrantfile')) | Set-Content -LiteralPath Vagrantfile
+
+- Start the virtual machine and login:
+
+.. code:: ps1
+
+    vagrant up
+    vagrant ssh
+
+You can now run ``dcos-docker`` commands.
```

Comment:
... and run code which uses the library. I'd change ``` ``dcos-docker`` ``` to `` :doc:`cli` `` - or at least try that to see if it is better, and maybe then add ``` :doc:`library` ```.